bazel/build-version.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Copyright 2020 The Verible Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Invoke bazel with --workspace_status_command=bazel/build-version.sh to
+# get this invoked and populate bazel-out/volatile-status.sh
+echo "GIT_HASH $(git log -n1 --date=short --format='"verible_%cd_%h"' || echo '"<unknown>"')"
```

Comment:
Can we add a `git-describe` output?

Reply:
Related question: do people still download source code distribution tarballs these days (not just git-clone)?  If so, those will need to be pre-packed with some stamp like `git-describe`.

Reply:
I don't like the git-describe output as it only contains a hash and a somewhat meaningless number generated from the number of commits.

A date, that contains information about roughly when to look for a particular commit happens is much nicer.

Reply:
I think if people compile from tar-file, they are on their own.

Reply:
git-describe is strictly worse than the string we have now. Would there be a good reason to include it ?

Reply:
Done.